.github/workflows/release.yaml

```diff
@@ -454,7 +464,9 @@ jobs:
           .\wingetcreate.exe update Coder.Coder `
             --submit `
             --version "${version}" `
-            --urls "${installer_url}|X64" `
+            --urls "${x64_installer_url}|X64" `
+            --urls "${x64_zip_url}|X64" `
+            --urls "${arm64_zip_url}" `
```

Comment:
How does this work? You can specify 3 URLs (two with the same `|X64` and it just works? That's a bit too magic for my tastes...

Reply:
Yes, it works like that
https://github.com/microsoft/winget-create/blob/main/doc/update.md#usage-examples

We do not need extra `--urls` flags. Thanks for pointing.